packages/gatsby-plugin-sitemap/src/__tests__/gatsby-node.js

```diff
@@ -1,21 +1,30 @@
-jest.mock(`fs`)
+jest.mock(`sitemap`, () => {
```

Comment:
Can you add tests for all functionalities like filters

Reply:
The filters are being tested in the `internals.js` tests (or will be)...what tests do you think should be here?